src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -926,6 +926,58 @@ private unsafe void WriteCounterToServer(DocumentsOperationContext context, Repl
             }
         }
 
+        private unsafe void WriteCountersToServer(DocumentsOperationContext context, ReplicationBatchItem item)
+        {
+            using (Slice.From(context.Allocator, item.ChangeVector, out var cv))
+                fixed (byte* pTemp = _tempBuffer)
+                {
+                    var requiredSize = sizeof(byte) + // type
+                                       sizeof(int) + // change vector size
+                                       cv.Size + // change vector
+                                       sizeof(short) + // transaction marker
+                                       sizeof(int) + // size of doc id
+                                       item.Id.Size +
+                                       sizeof(int) + // size of doc collection
+                                       item.Collection.Size + // doc collection
+                                       sizeof(int) // size of data
+                                       + item.Values.Size; // data
+
+                    if (requiredSize > _tempBuffer.Length)
+                        ThrowTooManyChangeVectorEntries(item);
+
+                    var tempBufferPos = 0;
+                    pTemp[tempBufferPos++] = (byte)item.Type;
```

Comment:
ensure the value is different from the value in 41

Reply:
done